.github/workflows/slsa3_builder.yml

```
@@ -41,6 +40,11 @@ on:
         required: false
         type: boolean
         default: true
+      config-file:
+        description: "The configuration file defining the builds"
```

```suggestion
        description: "The configuration file for the builder. A path within the calling repository."
```
WDYT to this which a) makes it clearer it's configuring the reusable action and b) makes it clearer the path is within the users repo.